lib/components/viewers/nearby/nearby-view.tsx

```diff
@@ -270,7 +270,7 @@ const mapStateToProps = (state: AppReduxState) => {
   const { nearby } = transitIndex
   const { entityId } = state.router.location.query
   return {
-    entityId,
+    entityId: entityId !== undefined ? decodeURIComponent(entityId) : undefined,
```

Comment:
Thanks!

Reply:
Does `entityId && decodeURIComponent(entityId)` cause issues?

Reply:
Should be fine. Refactored in https://github.com/opentripplanner/otp-react-redux/pull/1153/commits/aa9c864cb6327eea5b5ed37962229e40b99aabec.